internal/cli/deployment_list.go

```diff
@@ -177,28 +165,29 @@ func (c *DeploymentListCommand) Run(args []string) int {
 				completeTime = humanize.Time(t)
 			}
 
-			// Only display "Latest" report on Last deployment
-			var statusReportComplete string
-			if statusReportResp != nil {
-				if i == 0 { // Latest deployment
-					switch statusReport.Health {
-					case sdk.StatusReport_READY:
-						statusReportComplete = "✔"
-					case sdk.StatusReport_ALIVE:
-						statusReportComplete = "✔"
-					case sdk.StatusReport_DOWN:
-						statusReportComplete = "✖"
-					case sdk.StatusReport_PARTIAL:
-						statusReportComplete = "●"
-					case sdk.StatusReport_UNKNOWN:
-						statusReportComplete = "?"
-					}
-					if t, err := ptypes.Timestamp(statusReport.TimeGenerated); err == nil {
-						statusReportComplete = fmt.Sprintf("%s - %s", statusReportComplete, humanize.Time(t))
+			// Add status report information if we have any
+			statusReportComplete := "Unknown Status"
+			for _, statusReport := range statusReportsResp.StatusReports {
+				if deploymentTargetId, ok := statusReport.TargetId.(*pb.StatusReport_DeploymentId); ok {
+					if deploymentTargetId.DeploymentId == b.Id {
+						switch statusReport.Health.HealthStatus {
```

Comment:
Happy to see we’re switching on the outer `HealthStatus` rather than the SDK’s `Health` enum.